packages/react/src/components/Forms/DatePicker/index.scss

```diff
@@ -1,5 +1,10 @@
 @use 'src/utils';
 
+
+%shadow {
+  box-shadow: 0px 1px 4px rgba(0, 0, 0, 0.3);
+}
```

Comment:
Let's move it into utils then use it within : 
- `DatePicker/index.scss`
- `Popover/index.scss`
- `Select/index.scss`